### PR TITLE
Add MSSQL to documentation

### PR DESCRIPTION
**Pull request by crwhite21**
Added MSSQL to the documentation to reflect its support. Also note, postgres was added to dialectOptions support. Probably, should have been another commit/pull request. ¯\\_(ツ)_/¯

### CODE REVIEW

File: docs/api/sequelize.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ var sequelize = new Sequelize('mysql://localhost:3306/database', {})
 | [username=null] | String | The username which is used to authenticate against the database. |
 | [password=null] | String | The password which is used to authenticate against the database. |
 | [options={}] | Object | An object with options. |
-| [options.dialect='mysql'] | String | The dialect you of the database you are connecting to. One of mysql, postgres, sqlite and mariadb |
+| [options.dialect='mysql'] | String | The dialect you of the database you are connecting to. One of mysql, postgres, sqlite, mariadb, and mssql |
```
Comment (mickhansen) [inline docs/api/sequelize.md]: You shouldn't change `docs/api/*` directly, they are generated from code comments.

**Comment by mickhansen** [comment]
ping @crwhite21

**Comment by crwhite21** [comment]
@mickhansen I will submit a new pull request with the proper changes in lib/sequelize.js a bit later tonight. This guy is safe to close.

**Comment by mickhansen** [comment]
Merging for the `docs/` changes. The `docs/api` change won't really take effect though.

**Pull request merged**